app/views/inthenewsworks/show.html.erb

```diff
@@ -57,9 +57,9 @@
 <% end %>
 
 <%# TODO commentary stuff %>
```

Comment:
Remove this now? Maybe just run the `rails todo` thing or search for TODOs? There might be others that are ready to be taken out now :)

Reply:
I think this is outdated.  I had actually searched for "TODO" in the repo earlier, and I think that the only ones remaining are stuff outside of what I was working on here, although there definitely are some lingering.

Reply:
Okay, cool beans. I had seen a couple as I was going and knew some were later tossed but wasn't keeping close track of all of them :)